### PR TITLE
ROX-15147: Add CentralCertificate and ScannerCertificate cards to System Health

### PR DESCRIPTION
**Pull request by pedrottimark**
### Problems

Although central and scanner certificate have banners when expiration is within 2 weeks, it is consistent to include in System Health so:

1. It is possible to know the amount of time until expiration, similar to sensor certificate expiration on Clusters page (although with different mapping from time to color) so customers can plan ahead.

2. It becomes possible to download certificate on demand so customer can update certificates on demand in response to an internal security incident.

### Analysis

Similar to src/Containers/MainPage/Banners/CredentialExpiryBanner.tsx

### Solution

1. Add **Central certificate** and **Scanner certificate** cards at the **end** of System Health page, so cards which are more likely to have errors are less likely to require scrolling.

2. Move files from Components and utils folders into more meaningful folders.

3. Factor out reusable CardHeaderIcons.

4. Factor out common code into utils/credentialExpiry.ts file.

### Residue

1. Move icon to left in header of other cards.
2. Write integration tests.
3. Factor out external link as a reusable component.
4. Delete obsolete utils files after Ivan rewrites IntegrationHealth cards.

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui
3. `yarn start` in ui

### Manual testing

**NOTE**: A follow up pull request will move icon to left in header of other cards.

1. Visit /main/system-health

    * See GET /v1/credentialexpiry requests.
        credentialExpiry

    * Temporarily edit code to see Spinner while fetching initial request.
        Spinner

    * Temporarily edit code to see error message if GET /v1/credentialexpiry request fails.
        errorMessageFetching

    * Temporarily edit code to see warning if expiration date is in 14 days.
        warning

    * Temporarily edit code to see danger if expiration date is in 3 days.
        danger

    * Temporarily edit code to see sentence for non-administrator role if expiration date is in 3 days.
        danger_non-admin

    * Temporarily edit code to see absence of sentence for non-administrator role if expiration date is greater than 14 days.
        success_non-admin

    * Temporarily edit code to see disabled and spinner for **Download YAML** button.
        Spinner_Download_YAML

    * Temporarily edit code to see error message if POST /api/extensions/certgen request fails.
        errorMessageDownloading

    * Click external link to see **Reissuing internal certificates** docs page open in a new browser tab.

### Integration testing

1. `yarn cypress-open` in ui/apps/platform

    * credentialExpiry/credentialExpiry.test.js

### CODE REVIEW

File: ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js [hunk 1]
```diff
@@ -54,6 +55,12 @@ const SystemHealthDashboardPage = () => {
                     <GridItem span={12}>
                         <DeclarativeConfigurationHealthCard pollingCount={pollingCountFaster} />
                     </GridItem>
+                    <GridItem span={12}>
+                        <CertificateCard component="CENTRAL" pollingCount={pollingCountSlower} />
+                    </GridItem>
+                    <GridItem span={12}>
+                        <CertificateCard component="SCANNER" pollingCount={pollingCountSlower} />
+                    </GridItem>
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js:63]: @vjwilson
* Do we need conditional rendering to omit for cloud service?
* If yes, remind me how we are doing that?

Comment (vjwilson) [reply ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js:63]: 1. Yes, we will need conditional rendering to not show these in cloud service.
2. We do not have a way to that yet. Misha is coordinating an effort to have a generalized solution for this available sometime during 4.2 development cycle.

I think the added value for self-managed customers justifies adding as-is for now.

**Review by vjwilson — APPROVED** [review]
I answered your questions about Cloud Service version inline.

File: ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js [hunk 2]
```diff
@@ -3,11 +3,12 @@ import { Flex, FlexItem, Grid, GridItem, PageSection, Title } from '@patternfly/
...
 
+import CertificateCard from './CertificateHealth/CertificateCard';
 import ClustersHealthCards from './ClustersHealth/ClustersHealthCards';
+import DeclarativeConfigurationHealthCard from './DeclarativeConfigurationHealth/DeclarativeConfigurationHealthCard';
+import GenerateDiagnosticBundle from './DiagnosticBundle/GenerateDiagnosticBundle';
+import VulnerabilityDefinitionsHealthCard from './VulnerabilityDefinitionsHealth/VulnerabilityDefinitionsHealthCard';
 
-import DeclarativeConfigurationHealthCard from './Components/DeclarativeConfigurationHealthCard';
-import GenerateDiagnosticBundle from './Components/GenerateDiagnosticBundle';
-import VulnerabilityDefinitionsHealthCard from './Components/VulnerabilityDefinitionsHealthCard';
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js:10]: @ivan-degtiarenko
1. You might need to resolve merge conflict here when you rebase #6222
2. As we discussed on the call:
    * Please add your rewritten components into IntegrationHealth folder and delete Components folder
    * Also move anything still useful from utils into IntegrationHealth folder and delete utils folder

**Pull request merged**